python/PythonParser.g4

```diff
@@ -42,68 +42,102 @@ eval_input
     : testlist NEWLINE*
     ;
 
+stmt
+    : simple_stmt
+    | compound_stmt
+    ;
+
+//---------------- compound statement ----------------------------------------------------------------------------------
```

Comment:
What is the meaning of such comments? I don't think it makes sense here. So, remove it here and below.

Reply:
The purpose of the comments is to match a rules subset to a test sample. E.g. `funcdef.py` tests lines 99-132. Found it useful while writing tests (navigating through the grammar). Should I remove them anyway?